edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Core/Constants.cs

```diff
@@ -95,6 +95,12 @@ public static class Constants
 
         public const string EdgeletInitializationVectorFileName = "IOTEDGE_BACKUP_IV";
 
+        public const string PersistantVolumeNameKey = "PersistantVolumeName";
+
+        public const string StorageClassNameKey = "StorageClassName";
+
+        public const string PersistantVolumeClaimDefaultSizeKey = "PersistantVolumeClaimDefaultSizeInMb";
```

Comment:
Should these constants be defined here? They seem `k8s` specific.

Reply:
These are parameters that can be passed in as environment variables, and are used in Program.cs - would there be a better place to put them?


---
In reply to: [284486252](https://github.com/Azure/iotedge/pull/1199#discussion_r284486252) [](ancestors = 284486252)